src/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -120,6 +122,13 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
         >
           Close
         </ToolbarButton>
```

Comment:
The button can be part of the modal module and isLaunchModalOpen state could be handled there as well.
See https://github.com/openshift-metal3/facet/blob/29afdab0857021013d2217eda157b9f4b08eb462/src/components/clusters/clusterEventsModal.tsx